_posts/2023-05-26-cryostat-release.md

```diff
@@ -4,9 +4,12 @@ title: Cryostat 2.3.0 is Released!
 date: 2023-05-26
 synopsis: Cryostat 2.3.0 is here. Learn what's new in this release.
 author: Andrew Azores
-toc: true
 ---
 
+* auto-gen TOC:
```

Comment:
Should these auto-gen`d TOCs be put in the other posts as well?

Reply:
Sure, I can do that.